.github/workflows/e2e-conformance-trigger.yaml

```diff
@@ -10,10 +10,10 @@ jobs:
       fail-fast: false
       matrix:
         k8s_version: [ "1.23", "1.24", "1.25", "1.26", "1.27" ]
-    uses: ./.github/workflows/e2e.yaml
+    uses: ./.github/workflows/e2e-matrix.yaml
     with:
-      suite: Integration
       event_name: conformance
+      region: "us-west-2"
```

Comment:
Didn't we say that we were going to move this to a different region? I think we said `eu-west-1` or something similar?